etc/fetchers/fetch-sheets.ts

```diff
@@ -1,57 +1,24 @@
-const cheerio = require("cheerio");
-const fs = require("fs");
-const path = require("path");
-const fetch = require("cross-fetch");
+import {
+  allIsEmptyString,
+  getSlug,
+  toSnakeCase,
+  toTitleCase,
+} from "../../lib/string-utils";
 
-function toSnakeCase(text) {
-  return text.trim().toLowerCase().replace(" ", "_");
-}
-function toTitleCase(str) {
-  return str
-    .split(" ")
-    .map((word) => word.charAt(0).toUpperCase() + word.slice(1))
-    .join(" ");
-}
-function replaceSpacesWithCamelCase(str) {
-  return str.replace(/\s+/g, (s) => {
-    return s.substring(0, 1).toUpperCase() + s.substring(1);
-  });
-}
-function replaceSpecialCharacterWithSpace(str) {
-  return str.replace(/[^a-zA-Z0-9. ]/g, " ");
-}
-function removeSpaces(str) {
-  return str.replace(/\s+/g, "");
-}
-function convertToKebabCase(str) {
-  return str.replace(/([a-z])([A-Z])/g, "$1-$2").toLowerCase();
-}
-function getSlug(name, index) {
-  const kebabName = convertToKebabCase(
-    removeSpaces(
-      replaceSpecialCharacterWithSpace(replaceSpacesWithCamelCase(name)),
-    ),
-  );
-  return `${kebabName}-${index}`;
-}
-function allIsEmptyString(array) {
-  for (const item of array) {
-    if (item.length !== 0) {
-      return false;
-    }
-  }
-  return true;
-}
+import cheerio from "cheerio";
+import fetch from "cross-fetch";
+import fs from "fs";
+import path from "path";
```

Comment:
Import from 3rd parties are better at first line.
So, please move the import from `utils` after import from 3rd parties.

Reply:
Doing that will cause conflicts with [simple sort rules](https://github.com/lydell/eslint-plugin-simple-import-sort). Should we ignore it?

Reply:
See on the example:

https://github.com/lydell/eslint-plugin-simple-import-sort#example

The `react` always coming first, before the internal code.

Reply:
The other tips for node.js, you need to add a newline to seperate the native node.js API such as `fs` and `path` with any other libraries

Reply:
```ts
import fs from "fs";
import path from "path";

import cheerio from "cheerio";
import fetch from "cross-fetch";

// ...internal import
```

Reply:
It seems that the code from [here](https://github.com/kawalcovid19/wargabantuwarga.com/pull/172#discussion_r672801955) cannot be commited except if we skipped the pre-commit hooks.

The problem is caused by the pre-commit hooks which executes `eslint --cache --fix` which will automatically formats `fetch-sheets.ts` to the current code. Should we temporarily bypass the pre-commit hook for this commit?

Reply:
Just ignore it for now.

Maybe need to think to fix the linter.